repos/system_upgrade/el8toel9/actors/dotnet/actor.py

```diff
@@ -0,0 +1,35 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.libraries.common.rpms import has_package
+from leapp.models import Report, InstalledRPM
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+UNSUPPORTED_VERSIONS = [ '2.1', '3.0', '3.1', '5.0' ]
+
+class DotnetUnsupportedVersionsCheck(Actor):
+    """
+    Check for installed .NET versions that are no longer supported.
+    """
+
+    name = 'dotnet_unsupported_versions_check'
+    produces = (Report,)
```

Comment:
```suggestion
    consumes = (InstalledRPM,)
    produces = (Report,)
```

Reply:
@tmds This should fix your issue with report not being created.
Long story short - unless you specify that your actor consumes InstalledRPM, has_package(InstalledRPM, 'smth') check won't work for you.

Reply:
Thanks for fixing, and explaining the issue!